Mathlib/RingTheory/Nullstellensatz.lean

```diff
@@ -168,7 +168,7 @@ theorem isMaximal_iff_eq_vanishingIdeal_singleton (I : Ideal (MvPolynomial σ k)
   simpa only [map_sum ϕ, ϕ.map_mul, map_prod ϕ, ϕ.map_pow, ϕ.map_zero, hx] using congr_arg ϕ hp
 
 /-- Main statement of the Nullstellensatz -/
-@[simp]
+-- Can't be `simp` due to simpNF timeout on LHS (failed to synthesize `I.IsPrime`)
```

Comment:
This seems like a regression. Do you understand how this is connected to the rest of the PR?
At the face of it, this lemma looks like a good simp lemma.

Reply:
Oh, I made an adjustment in #21544 that made this no longer necessary, but I forgot to revert this change.